HerPublicWebsite/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Disallow: /
```

Comment:
This preventing ALL indexing for the site, which doesn't feel quite right. We'd want the public facing side (HER equivalent of frontdoor?) to be indexable I assume

Reply:
Exactly. We'll disallow their version of the portal.